docs/samples/Microsoft.ML.Samples/Program.cs

```diff
@@ -1,12 +1,43 @@
-﻿using Samples.Dynamic;
+﻿using System;
+using System.Reflection;
+using Samples.Dynamic;
 
 namespace Microsoft.ML.Samples
 {
-    internal static class Program
+    public static class Program
     {
-        static void Main(string[] args)
+        public static void Main(string[] args)
         {
             CalculateFeatureContribution.Example();
+
+            if (args.Length > 0 && args[0] == "-runall")
+                RunAll();
+        }
+
+        internal static void RunAll()
+        {
+            foreach (var type in Assembly.GetExecutingAssembly().GetTypes())
+            {
+                var method = type.GetMethod("Example", BindingFlags.NonPublic | BindingFlags.Public
+                    | BindingFlags.Static | BindingFlags.FlattenHierarchy);
+
+                if (method != null)
```

Comment:
>if (method != null) [](start = 16, length = 19)

let's also handle the case where (method == null), which are all the cases where we don't conform with pattern below:
public static Example()

Reply:
@shmoradims there are types not related to sample classes and they don't have example methods and there is nothing we can do about it, i.e throw exceptions. I see your point you want to catch samples that are not following the pattern but then there is that.